rr_common/src/image_pcl_converter/image_pcl_converter.cpp

```diff
@@ -64,7 +64,7 @@ void transformedImageCB(const sensor_msgs::ImageConstPtr& msg, const string& top
     sensor_msgs::PointCloud2 cloud_msg;
     pcl_conversions::fromPCL(cloud_pc2, cloud_msg);
     cloud_msg.header.frame_id = "base_footprint";
-    cloud_msg.header.stamp = ros::Time::now();
+    cloud_msg.header.stamp = msg->header.stamp;
```

Comment:
This is something we should do more of / continue to do going forward. If we are processing timestamped sensor data, the time we care about for the result is probably when the sensor made the measurement, not when the processing happened